st2common/st2common/services/workflows.py

```diff
@@ -819,6 +826,13 @@ def request_next_tasks(wf_ex_db, task_ex_id=None):
         LOG.info(msg, wf_ac_ex_id, str(iteration), task_ex_db.task_id)
         LOG.debug('[%s] %s', wf_ac_ex_id, conductor.serialize())
         next_tasks = conductor.get_next_tasks()
+
+        if next_tasks and execution_id:
+            output_db = ex_db_access.ActionExecutionOutput.get(execution_id=execution_id)
```

Comment:
See my comment here - https://github.com/StackStorm/st2/pull/4529/files#r253811547.

This also wouldn't work because there could be thousands of ``ActionExecutionOutputDB`` objects which refer to that particular ``execution_id`` and ``get()`` only works for unique values aka if there is only a single entry for that value in the database.